scripts/flow.sh

```diff
@@ -33,7 +33,7 @@ delete() {
     for id in $environment_ids; do
         if [ "$id" != "api" ]; then
             echo -e $BULLET "$id"
-            l0 environment delete $id --wait > /dev/null &
+            l0 environment delete $id > /dev/null
```

Comment:
[nit] you could probably keep the `&` if you added a `wait` command after the for loop.

Reply:
We would need to capture every PID that is spawned. This is a simpler solution because it simply just executes each delete and waits for the environment to delete before moving on to the next one. I'm a little concerned about backgrounding tasks like this. I'm not sure we'd want to for instance spawn 100 background tasks at the same time to handle cleanup.